### PR TITLE
Fix heroku config file error and set default value

### PR DESCRIPTION
**Pull request by bananaappletw**
Fix  heroku config file error and set default value.
Easier to deploy on heroku.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,11 @@ You can quickly setup a sample heroku hackmd application by clicking the button
 
 [![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy)
 
+You have to setup the minimal configuration
+
+- App Name ( Ex: hackmd )
+- HMD_DOMAIN ( Ex: hackmd.herokuapp.com )
```
Comment (Yukaii) [inline README.md]: I do prefer to let Heroku handle domain itself. I'm not setting `HMD_DOMAIN` value, and both heroku domain(xxx.herokuapp.com) and my custom domain work. 😄

File: app.json [hunk 2]
```diff
@@ -36,17 +36,19 @@
             "required": false
         },
         "HMD_ALLOW_ORIGIN": {
```
Comment (jackycute) [inline app.json]: So here actually a mistaken from @Yukaii 
It should be `HMD_PORT` instead of `HMD_ALLOW_ORIGIN `.

File: app.json [hunk 3]
```diff
@@ -36,17 +36,19 @@
...
-            "description": "web app port",
+            "description": "domain to allow cross origin request",
```
Comment (jackycute) [inline app.json]: don't change this

File: app.json [hunk 4]
```diff
@@ -36,17 +36,19 @@
...
             "required": false,
-            "value": "80"
+            "value": "localhost"
```
Comment (jackycute) [inline app.json]: don't change this

**Comment by bananaappletw** [comment]
Sorry, I misunderstood the meaning of config file.

**Pull request closed without merging**